Author: Philip Boulain <philip.boulain@smoothwall.net>
Description: Fix build under Debian hardening flags.
 Add constant format string to use of printf().
--- a/bp_util.c
+++ b/bp_util.c
@@ -1214,7 +1214,7 @@
 
     for (i=0;bp_cap_array[i].flag;i++) {
         if (bpctl_cmd.status & bp_cap_array[i].flag)
-            printf(bp_cap_array[i].desc);
+            printf("%s", bp_cap_array[i].desc);
     }
 }
 
